packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -24,7 +55,52 @@ export class Snapshot {
     // TODO: CDX-347 Display Expanded preview
   }
 
-  async delete() {
-    // TODO: CDX-359: Delete snapshot once previewed
+  private get lastestReport(): ResourceSnapshotsReportModel {
+    if (this.model.reports === undefined) {
+      throw new Error(
+        `No detailed report found for the snapshot ${this.snapshotId}`
+      );
+    }
+    return this.model.reports.slice(-1)[0];
```

Comment:
Anything we might want to handle as far as reports array being empty ?
Is it even possible ?

Reply:
Probably a question to the backend team.
I included the `includeReport` query parameter on every GET request to the snapshot API. 

I'll ask for confirmation